CONCEPTS.md

```diff
@@ -57,25 +57,25 @@ In this case, the translations are added as-is in a JSON file:
     "BTN_MARK": "Mark this question",
 ```
 
-To use this translation value, you can use the `_i18n_` helper, e. g in a HTML template:
+To use this translation value, you can use the `/code/kuusi/web/templates/tags/_i18n_` helper, e. g in a HTML template:
```

Comment:
The helper is the function `_i18n_()` within `web_extras.py`, but there is no file called `_i18n_`